scheds/rust/scx_layered/src/main.rs

```diff
@@ -1225,6 +1227,7 @@ impl<'a> Scheduler<'a> {
         // Open the BPF prog first for verification.
         let mut skel_builder = BpfSkelBuilder::default();
         skel_builder.obj_builder.debug(opts.verbose > 1);
+        init_logging(None);
```

Comment:
Plus, users would also be able to use a more descriptive name than `init_logging()`.